pom.xml

```diff
@@ -87,8 +87,26 @@
             <artifactId>pride-jaxb</artifactId>
             <version>${uk.ac.ebi.pride.jaxb-pride-jaxb.version}</version>
         </dependency>
-
-
+        <dependency>
```

Comment:
@fcyu the current library is not using any of those libraries. Why do you want to add them to this library pom file.

Reply:
It looks like JmzReader relays on javax.xml.bind and javax.activation, which are not included in Java 9. 

You may reproduce the errors in using JmzReader under Java 9. You may also refer to [https://stackoverflow.com/questions/43574426/how-to-resolve-java-lang-noclassdeffounderror-javax-xml-bind-jaxbexception-in-j](https://stackoverflow.com/questions/43574426/how-to-resolve-java-lang-noclassdeffounderror-javax-xml-bind-jaxbexception-in-j) for some deatils.

Thanks

Reply:
@fcyu Thanks for reporting this. I will check the code and see if I can remove those dependencies. We are in a major refactoring and we would like to avoid as much as possible the introduction of more dependencies. I'll review the code today and be back to you soon

Reply:
@ypriverol Thanks. It would be great if you can remove those dependencies because they are deprecated since Java 9 and they may cause more issues in the future.